tools/ci/taskcluster-run.py

```diff
@@ -23,7 +23,11 @@ def get_browser_args(product, channel, artifact_path):
         return ["--install-browser", "--processes=12"]
     if product == "chrome" or product == "chromium":
         # Taskcluster machines do not have GPUs, so use software rendering via --enable-swiftshader.
-        return ["--enable-swiftshader", "--install-browser", "--install-webdriver"]
+        args = ["--enable-swiftshader"]
+        # Chrome Stable-Dev are still installed in tools/ci/run_tc.py
+        if channel == "nightly" or channel == "canary":
+            args.extend(["--install-browser", "--install-webdriver"])
```

Comment:
Wait, don't we need `--install-webdriver` for Stable-Dev? And for CfT, we shouldn't need it, because we get it from the same package?

Reply:
ChromeDriver is still always a separate download, although we maintain the download url to match exactly the browser binary version downloaded.

To be honest, I'm not certain, but if I had to guess, I would assume that the default behavior is to prompt webdriver installation if a webdriver is not found. So that means `--install-webdriver` is implicit as long as there is no other found webdriver, and as long as we're passing the `--yes` flag, it will just accept that installation.

Reply:
Hmm, so it sound like things work, but maybe we're double downloading the chromedriver binary in the CfT case. Can you file an issue if you think that's the case? Then feel free to land this.